q/unzip/unzip.q

```diff
@@ -0,0 +1,695 @@
+.finos.dep.include"../util/util.q"
+
+
+// Utilities
+
+// Read bytes from either a file or a byte vector.
+// @param x hsym or bytes
+// @param y offset
+// @param z length
+// @return z bytes from x, starting at y
+.finos.unzip.priv.bytes:{$[-11h=t:type x;read1(x;y;z);4h=t;x y+til z;'`type]}
```

Comment:
How does the `til` method of slicing an array compare to take+drop for large sizes? Or is this supposed to be used for small sizes only?

Reply:
i haven't actually timed it, but take+drop involves an intermediate list which is longer than the index list created by `til`, so i wouldn't expect it to be significantly better, if any